src/components/banner/banner.jsx

```diff
@@ -1,137 +1,151 @@
+                style={{
+                    boxShadow: `0px 0px 5px ${theme.primary}`,
+                    backgroundColor: theme.background,
+                    color: theme.textLight
+                }}
+            >
+                <div class={style.content}>
+                    <div
+                        class={style.message}
+                        ref={el => this.messageRef = el}
+                    >
+                        <div class={style.info}>
+                            <div class={style.title} style={{color: theme.text}}>
+                                <LocalLabel localizeKey='title'>Ads help us run this site</LocalLabel>
+                            </div>
+                            <LocalLabel localizeKey='description'>
+                                When you visit our site, pre-selected companies may access and use certain information
+                                on your device to serve relevant ads or personalized content.
+                            </LocalLabel>
+                            <div class={style.options}>
+                                <div
+                                    class={[style.option, selectedPanelIndex === PANEL_COLLECTED && isExpanded ? style.expanded : ''].join(' ')}>
+                                    <a onClick={this.handleInfo(PANEL_COLLECTED)} style={{color: theme.textLink}}>
+                                        <ChevronIcon/>
+                                        <LocalLabel localizeKey='links.data.title'>Information that may be
+                                            used.</LocalLabel>
+                                    </a>
 
-		return (
-			<div
-				ref={el => this.bannerRef = el}
-				class={style.banner}
-				style={{bottom: `${bannerBottom}px`}}
-			>
-				<div class={style.content}>
-					<div
-						class={style.message}
-						ref={el => this.messageRef = el}
-					>
-						<div class={style.title}>
-							<LocalLabel localizeKey='title'>Ads help us run this site</LocalLabel>
-						</div>
-						<LocalLabel localizeKey='description'>
-						When you visit our site, pre-selected companies may access and use certain information on your device to serve relevant ads or personalized content.
-						</LocalLabel>
-						<div class={style.info}>
-							<a onClick={this.handleInfo(PANEL_COLLECTED)}>
-								<ChevronIcon class={[style.expand, selectedPanelIndex === PANEL_COLLECTED && isExpanded ? style.expanded : ''].join(' ')}  />
-							</a>
-							<LocalLabel localizeKey='links.data.title'>Information that may be used.</LocalLabel>
-							<a onClick={this.handleInfo(PANEL_PURPOSE)}>
-								<ChevronIcon class={[style.expand, selectedPanelIndex === PANEL_PURPOSE && isExpanded ? style.expanded : ''].join(' ')}  />
-							</a>
-							<LocalLabel localizeKey='links.purposes.title'>Purposes for storing information.</LocalLabel>
+                                    <LocalLabel localizeKey='links.data.description'>
```

Comment:
It look likes the wrapper  `<div  class={style.optionDetails} style={{color: theme.textLight}}>` is missing around this tag. It is not collapsed by default.